x-pack/plugins/secops/public/lib/clipboard/clipboard.tsx

```diff
@@ -63,7 +63,7 @@ export class Clipboard extends React.PureComponent<Props, State> {
 
     return (
       <>
-        <div data-test-subj="clipboard" onClick={this.onClick}>
+        <div role="button" data-test-subj="clipboard" onClick={this.onClick}>
```

Comment:
note: This would probably be better later on to be a button. The linter complained with this error:

`react-a11y-event-has-role`
https://github.com/joaovieira/tslint-react-a11y
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role

Simplest fix for now is to add the role of that of a button.